core-scala/src/test/scala/com/baeldung/scala/partialfunctions/SwapIntegerSignUnitTest.scala

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.scala.partialfunctions
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+
+class SwapIntegerSignUnitTest {
+
```

Comment:
As per guidelines, we should be using ScalaTest for all unit tests rather than JUnit

Reply:
Hi @KevinGilmore , sure I am going to change it. 

However, I see all the other example are using JUnit. Is it a new guideline?

Reply:
done